index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> ULTRA-PALLOZA</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/6a7b1e2152.js" crossorigin="anonymous"></script>
+</head>
+<body>
+<!-- HOME PAGE -->
+<!-- HAMBURGER-WINDOW start-->
+
+<div  class="hambu-wind not-active"> 
+  <div class="x-btn"> 
+    <button type="btn-close" id="close">
+      <img src="/bg-img/blackclose.png" alt="x">
+    </button>
+  </div>
+  <a class="pop-window" href="/about.html">About</a>
+  <a class="pop-window" href="#about">Program</a>
+  <a class="pop-window" href="#footer"> Join</a>   
+  <a class="pop-window" href="#footer"> Sponsors</a>  
+  <a class="pop-window" href="#footer"> News</a>  
+</div>
+
+<!-- HAMBURGER-WINDOW end-->
+
+  <!-- SECTION HERO -->
+<section id="section-hero">
+  <div id="hero-div">
+    <div class="social-media">
+      <i class="fa-brands fa-square-facebook fa-2x"></i>
+      <i class="fa-brands fa-square-twitter fa-2x"></i>
+    </div>
+    <nav id="navbar">
+      <div id="nav-content"> 
+        <a id="ham-link" >
+        <img id="ham-btn" src="/bg-img/Hamburger_icon.svg.png" alt="hamburger-icon">
+        </a>
+        <a id="logo-mobile"> <img id="logo-btn-show" src="/bg-img/logo.png" alt="logo-icon"> </a>
+        <a id="logo-link" href="/index.html">
+          <img id="logo-btn" src="/bg-img/logo.png" alt="logo-icon">
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
+    <div id="hero-bg"> 
+    <div class="title">      
+      <h1> <span class="margin-none">Music is life!</span> <br> GET READY FOR THE ULTIMATE FESTIVAL WINTER 2023</h1>
+    </div>
+    <div class="paragraph"> Lolla takes over historic Grant Park in Downtown Chicago, alongside the beautiful Chicago skyline and the shores of Lake Michigan. Take advantage of interactive areas 
+      with giveaways, merch and more!</div>
+    <div class="date">SAVE THE DATE! MARCH 2-5-2023
+      <p>@ Chicago, Illinois, USA.</p>
+    </div>
+  </div>
+  </div>
```

Comment:
- [ ] As I mentioned in the previous comment, according to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), so I suggest to use the same font-size and to align the content to the left (instead of center it) to match the original design layout 👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/197104950-a63eecad-ea2f-4497-b92a-b34ece533cb3.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/197104919-0296be74-0062-4047-a8a3-6457c8d5a0a3.PNG" width="500" /> |



> It's ok to customize the content and use different images but you should follow the given design template. There will be projects in the curriculum where you will be able to show your creative designs. But for this project we expect you to clone the given design.